pom.xml

```diff
@@ -110,7 +110,7 @@
                        <Bundle-Name>NuProcess</Bundle-Name>
                        <Export-Package>com.zaxxer.nuprocess</Export-Package>
                        <Export-Package>com.zaxxer.nuprocess.codec</Export-Package>
-                       <Import-Package>com.sun.*,sun.misc.*</Import-Package>
+                       <Import-Package>com.sun.*,sun.misc.*,org.slf4j</Import-Package>
```

Comment:
This change seems unnecessary.  I am ready to accept this pull request after this is removed.